Short project name causes failure in flake8-isort

## What happened?

`flake8-isort` fails on two files if the project name is too short.

For my project name with 8 characters, it fails in `[project_apps_dir]/users/tests/test_views.py` and `[project_apps_dir]/users/urls.py`

_Originally posted by @alexche77 in https://github.com/pydanny/cookiecutter-django/issues/2502#issuecomment-638589822_

## What should've happened instead?

No flake8 failure. All style failures should either not be there, or be fixed by pre-commit hooks.

## Steps to reproduce

```
# cookiecutter gh:pydanny/cookiecutter-django --no-input project_slug=abcdefgh
# git init
# pre-commit install
# git add .
# git commit
# (pre-commit fix cycle for trailing-whitespace, end-of-file-fixer and black)
# git commit
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check Yaml...............................................................Passed
black....................................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

abcdefgh/users/urls.py:3:1: I001 isort found an import in the wrong position
abcdefgh/users/urls.py:4:1: I001 isort found an import in the wrong position
abcdefgh/users/urls.py:5:1: I001 isort found an import in the wrong position
abcdefgh/users/urls.py:6:1: I001 isort found an import in the wrong position
abcdefgh/users/urls.py:7:1: I001 isort found an import in the wrong position
abcdefgh/users/tests/test_views.py:8:1: I001 isort found an import in the wrong position
abcdefgh/users/tests/test_views.py:9:1: I001 isort found an import in the wrong position
abcdefgh/users/tests/test_views.py:10:1: I001 isort found an import in the wrong position
abcdefgh/users/tests/test_views.py:11:1: I001 isort found an import in the wrong position
abcdefgh/users/tests/test_views.py:12:1: I001 isort found an import in the wrong position
```

## Suggested solution

Either use a different import clause depending on the project name length (ugly?), or put `isort`as a separate hook in pre-commit (before `flake8`) so it will auto-correct before `flake8` runs. I tested the latter and it works.

--- a/CONTRIBUTORS.rst
+++ b/CONTRIBUTORS.rst
@@ -141,6 +141,7 @@ Listed in alphabetical order.
   Isaac12x                 `@Isaac12x`_
   Ivan Khomutov            `@ikhomutov`_
   James Williams           `@jameswilliams1`_
+  Jan Fabry                `@janfabry`_
   Jan Van Bruggen          `@jvanbrug`_
   Jelmer Draaijer          `@foarsitter`_
   Jerome Caisip            `@jeromecaisip`_
@@ -331,6 +332,7 @@ Listed in alphabetical order.
 .. _@ikkebr: https://github.com/ikkebr
 .. _@Isaac12x: https://github.com/Isaac12x
 .. _@iynaix: https://github.com/iynaix
+.. _@janfabry: https://github.com/janfabry
 .. _@jangeador: https://github.com/jangeador
 .. _@jazztpt: https://github.com/jazztpt
 .. _@jcass77: https://github.com/jcass77
@@ -392,7 +394,7 @@ Listed in alphabetical order.
 .. _@siauPatrick: https://github.com/siauPatrick
 .. _@sladinji: https://github.com/sladinji
 .. _@slafs: https://github.com/slafs
-.. _@sorasful:: https://github.com/sorasful
+.. _@sorasful: https://github.com/sorasful
 .. _@ssteinerX: https://github.com/ssteinerx
 .. _@step21: https://github.com/step21
 .. _@stepmr: https://github.com/stepmr

--- a/{{cookiecutter.project_slug}}/.pre-commit-config.yaml
+++ b/{{cookiecutter.project_slug}}/.pre-commit-config.yaml
@@ -15,6 +15,11 @@ repos:
     hooks:
       - id: black
 
+  - repo: https://github.com/timothycrosley/isort
+    rev: 4.3.21
+    hooks:
+      - id: isort
+
   - repo: https://gitlab.com/pycqa/flake8
     rev: 3.8.3
     hooks:

